src/pages/group/group.ts

```diff
@@ -51,6 +52,27 @@ export class GroupPage {
 		this.arePlayersSelected = playersSelected.length > 0;
 	}
 
+	/**
+	 * Allows the point listed to become editabl.e
```

Comment:
minor typo: editable. Also this should be updated to say it can toggle the edit functionality.
You might rename the function toggleEditMode or something.